WordPress/UITestsFoundation/Screens/Media/MediaPickerAlbumScreen.swift

```diff
@@ -26,20 +39,10 @@ public class MediaPickerAlbumScreen: ScreenObject {
             index += 1
         }
 
-        app.buttons["SelectedActionButton"].tap()
-    }
-
-    func insertSelectedImage() {
-        app.buttons["SelectedActionButton"].tap()
+        selectedActionButton.tap()
     }
 
     public static func isLoaded(app: XCUIApplication = XCUIApplication()) -> Bool {
-        // Check if the media picker is loaded as a component within the editor
-        // and only return true if the media picker is a full screen
-        if app.navigationBars["Azctec Editor Navigation Bar"].exists {
-            return false
-        }
-
```

Comment:
this navigation bar doesn't exist on the app anymore so this condition can be removed